toolchain/check/node_block_stack.h

```diff
@@ -44,7 +44,7 @@ class NodeBlockStack {
   }
 
   // Returns the top node block, allocating one if it's still invalid.
-  auto PeekForAdd() -> SemIR::NodeBlockId;
+  auto PeekForAdd(int depth = 0) -> SemIR::NodeBlockId;
```

Comment:
With the edits, the comment should be updated since it doesn't always return the top node block.

Reply:
Done.